src/main/java/org/candlepin/async/JobManager.java

```diff
@@ -164,6 +164,8 @@ public Job newJob(TriggerFiredBundle bundle, Scheduler scheduler) throws Schedul
         @Override
         public void execute(org.quartz.JobExecutionContext context) throws org.quartz.JobExecutionException {
             String jobKey = context.getJobDetail().getKey().getName();
+            Principal principal = new JobPrincipal(SystemPrincipal.NAME + "." + jobKey);
+            ResteasyContext.pushContext(Principal.class, principal);
 
             try {
                 log.trace("Queuing job: {}", jobKey);
```

Comment:
I know this is unrelated to your work, and is incredibly minor, but can we use this opportunity to change the trace line to read "Queuing scheduled job: {}"?

Reply:
Sounds good to me! "Queuing scheduled job" makes more sense for this tracer